.azure-pipelines/weekly.yml

```diff
@@ -0,0 +1,22 @@
+schedules:
+ - cron: "0 23 * * 0"
+   displayName: Weekly CI (Sunday night UTC)
+   always: true
+   branches:
+     include:
+     - master
+
+variables:
+  PIP_CACHE_DIR: $(Pipeline.Workspace)/.cache/pip
+  DDEV_COLOR: 1
+
+jobs:
+- template: './templates/test-all-checks.yml'
+  parameters:
```

Comment:
`test-all-checks` has a single parameter called `pip_cache_config`, I don't think this will work. You'll have to add the relevant parameters to all the templates one by one until you reach `run-tests.yaml`.

Your `weekly.yml` file calls `test-all-checks.yml`, which calls `test-all.yml` which calls `test-single-linux.yml` (and `test-single-windows.yml` but we can ignore), which finally calls `run-tests.yml`.

The option will have to propagate up to there.

Reply:
Also can the file be named to reflect that this pipeline is only about checking "latest_metrics" ?

Reply:
> Also can the file be named to reflect that this pipeline is only about checking "latest_metrics" ?

Can do that, rationale for the name was possibly adding additional tests in the future on a weekly basis, but we can rename if we ever add them.

Reply:
> `test-all-checks` has a single parameter called `pip_cache_config`, I don't think this will work. You'll have to add the relevant parameters to all the templates one by one until you reach `run-tests.yaml`.

Ok, makes sense - I thought it might propagate defined parameters on the way down, but guess that doesn't work.

In that case, will just update to explicitly run the tests against known checks.